modules/run-consul/README.md

```diff
@@ -65,6 +65,12 @@ The `run-consul` script accepts the following arguments:
 * `data-dir` (optional): The path to the Consul config folder. Default is to take the absolute path of `../data`, 
   relative to the `run-consul` script itself.
 * `user` (optional): The user to run Consul as. Default is to use the owner of `config-dir`.
+* `enable-gossip-encryption` (optional): Enable encryption of gossip traffic between nodes. If set, you must also specify `gossip-encryption-key`.
+* `gossip-encryption-key` (optional): The key to use for encrypting gossip traffic. Must be specified with `enable-gossip-encryption`.
+* `enable-rpc-encryption` (optional): Enable encryption of RPC traffic between nodes. Must also specify `ca-file-path`, `cert-file-path` and `key-file-path`.
+* `ca-file-path` (optional): Path to the CA file used to verify outgoing connections. Must be specified with `enable-rpc-encryption`, `cert-file-path` and `key-file-path`.
+* `cert-file-path` (optional): Path to the certificate file used to verify incoming connections. Must be specified with `enable-rpc-encryption`, `ca-file-path`, and `key-file-path`.
+* `key-file-path` (optional): Path to the certificate key used to verify incoming connections. Must be specified with `enable-rpc-encryption`, `ca-file-path` and `cert-file-path`.
```

Comment:
I'm guessing the cert/key aren't used to "verify" incoming connections, but to encrypt them, right?

Reply:
Mmmhh... actually the official documentation uses "verify", and in fact the configuration fields are called `verify_ougoing` and `verify_incoming`, https://www.consul.io/docs/agent/encryption.html. I'm not so sure what verb to use...

Reply:
Well, I guess it's actually both verify and encrypt, depending on whether `verify_outgoing` and/or `verify_incoming` are set, and depending on whether you're looking at the servers or clients. In general, the cert/key is used to encrypt and/or sign a connection. If `verify_outgoing` is set, then clients will check that the servers they are talking to are using certs to encrypt and sign the connection. If `verify_incoming` is set, then servers will check that clients are using certs to encrypt and sign the connection.

Reply:
I think the technical answer is they can be used for both encryption and verification, but using the lingo from the official docs is fine.